test/benchee/configuration_test.exs

```diff
@@ -9,26 +9,29 @@ defmodule Benchee.ConfigurationTest do
 
   @default_config %Configuration{}
 
-  describe ".init/1" do
-    test "it crashes for values that are going to be ignored" do
+  describe "init/1" do
+    test "crashes for values that are going to be ignored" do
       assert_raise KeyError, fn ->
         init(runntime: 2)
       end
     end
 
-    test "it converts input keys to strings" do
+    test "converts maps to lists and input keys to strings" do
```

Comment:
input maps to lists, not all maps right? Funnily we do both, we normally convert options given to maps :grin: I hope this doesn't become a weird to track down bug here